beamer/tests/health/test_notify.py

```diff
@@ -0,0 +1,32 @@
+from beamer.health.notify import NotificationState
+
+
+def test_value_is_set(tmpdir):
+    state = NotificationState(tmpdir)
+
+    state.update("a", "RequestExpired")
+
+    assert state.is_set("a", "RequestExpired") is True
+    assert state.is_set("b", "RequestExpired") is False
```

Comment:
Please do not compare directly to literal booleans (PEP8):

```suggestion
    assert state.is_set("a", "RequestExpired")
    assert not state.is_set("b", "RequestExpired")
```